src/components/Input/Input.test.tsx

```diff
@@ -3,82 +3,157 @@ import * as Enzyme from "enzyme";
+                placeholder={"Input Placeholder"}
+                type={InputTypes.text}
             ></Input>
         );
     });
 
-    it("Renders label for inputId", () => {
-        expect(container.find("label").exists()).to.equal(true);
-        expect(container.find("label").prop("htmlFor")).to.equal("input");
-    });
-    it("Renters TextField", () => {
+    it("Renters Input", () => {
         expect(container.find("input").exists()).to.equal(true);
     });
-    it("Does not render helper if none is passed", () => {
-        expect(container.find("HelperErrorText").exists()).to.equal(false);
+
+    it("Renders aria-label", () => {
+        expect(container.find("input").prop("aria-label")).to.equal(
+            "Input Label"
+        );
+    });
+
+    it("Renders placeholder text", () => {
+        expect(container.find("input").props().placeholder).to.equal(
+            "Input Placeholder"
+        );
+    });
+
+    it("Adds aria-required prop if input is required", () => {
+        expect(container.find("input").prop("aria-required")).to.equal(true);
     });
 });
 
-describe("Input with Helper and Error", () => {
+describe("Input with Label", () => {
     let container;
     before(() => {
         container = Enzyme.mount(
-            <Input
-                labelId="label-input"
-                labelContent={<>Label</>}
-                inputId="input"
-                isRequired={true}
-                helperContentOpts={{
-                    id: "input-helper",
-                    content: <>This is some helpful text</>,
-                }}
-                errorContentOpts={{
-                    id: "input-error",
-                    content: <>Error Message Lorem Ipsum</>,
-                }}
-                showError={true}
-            ></Input>
+            <>
+                <Label htmlFor="inputID" id={"label"}>
+                    Input Label
+                </Label>
+                <Input
+                    id="inputID"
+                    labelId="label"
+                    ariaLabel="Input Label"
+                    helperTextId={"helperText"}
+                    required={true}
+                    placeholder={"Input Placeholder"}
+                    type={InputTypes.text}
+                ></Input>
+                <HelperErrorText isError={false} id={"helperText"}>
+                    Helper Text Content
+                </HelperErrorText>
+            </>
         );
     });
 
-    it("Renders label for inputId", () => {
-        expect(container.find("label").exists()).to.equal(true);
-        expect(container.find("label").prop("htmlFor")).to.equal("input");
-    });
-    it("Renters TextField", () => {
+    it("Renders Input", () => {
         expect(container.find("input").exists()).to.equal(true);
     });
-    it("Renders Helper", () => {
-        expect(container.find("#input-helper").exists()).to.equal(true);
+
+    it("Label points to its associated input", () => {
```

Comment:
This test can be removed because it's just testing `htmlFor="inputID"` which is just html functionality.